### DISCUSSION TITLE
How to embed an iframe?

### DISCUSSION THREAD

**Discussion by HemuManju** [writer/writer-framework#56] [General]
Hello, 

Thank you for this wonderful package. 

I trying to embed a iframe using HTML element, but without success. Can anyone please provide a minimal example of embedding an iframe?

Thanks in advance!

**Comment by ramedina86** [comment]
Hello, thank you! I'm glad you're enjoying Streamsync.

Embedding an iframe should be straightforward, it's just a matter of creating an _HTML Element_ component, setting the Element property to `iframe` and the `Attributes` to an object containing the `src` property.

An example of `Attributes` for an `iframe` could be:

```js
{
"src":"https://www.youtube.com/embed/JZAWl-pyivk"
}
```

Take into account that in many cases web servers will prevent their content from being embedded anywhere; there's not much you can do if that's the case. But this solution should work whenever a standard `iframe` works.

**Reply by HemuManju** [reply]
Thank you!
